plugin/src/main/java/hudson/plugins/findbugs/FindBugsDescriptor.java

```diff
@@ -1,6 +1,7 @@
 package hudson.plugins.findbugs;
 
 import hudson.Extension;
+
```

Comment:
Right, it is a good idea to look at your own diff to ensure that everything in it serves some purpose!